ci-operator/step-registry/gather/aws-console/gather-aws-console-commands.sh

```diff
@@ -39,7 +41,17 @@ fi
 
 aws --version
 
-REGION="$(jq -r .aws.region "${SHARED_DIR}/metadata.json")"
+# Special setting for C2S/SC2S, running aws command in source region (us-east-1)
+if [ "${CLUSTER_TYPE}" == "aws-c2s" ] || [ "${CLUSTER_TYPE}" == "aws-sc2s" ]; then
+  source_region=$(jq -r ".\"${REGION}\".source_region" "${CLUSTER_PROFILE_DIR}/shift_project_setting.json")
+  REGION=$source_region
+  if [ ! -f "${SHARED_DIR}/unset-proxy.sh" ]; then
+    echo "ERROR, unset-proxy.sh does not exist, exit now."
+    exit 1
+  fi
+  source "${SHARED_DIR}/unset-proxy.sh"
```

Comment:
I don't understand what this block does. Why do we need to unset proxy?

Reply:
In C2S/SC2S, connecting to proxy means _we are in C2S/SC2S network_, and there is no internet connection in it, so, it will cause the following command `aws ec2 get-console-output` fail.

We are using a SHIFT emulator on `us-east-1` instead of _real_ C2S/SC2S regions, so in this step we are operating instances on `us-east-1`, that's why I need to get source region from SHIFT setting:
```
source_region=$(jq -r ".\"${REGION}\".source_region" "${CLUSTER_PROFILE_DIR}/shift_project_setting.json")
REGION=$source_region
```